test/verify/check-system-realms

```diff
@@ -85,6 +85,7 @@ ExecStart=/bin/true
 @testlib.skipDistroPackage()
 class CommonTests:
     @testlib.timeout(900)
+    @testlib.skipImage("fails to leave the domain", "debian-*")
```

Comment:
Do we have an issue for this?

Reply:
Not yet, so far just an entry on my TODO list, as it's not a regression.